components/raftstore/src/store/fsm/store.rs

```diff
@@ -2604,6 +2602,22 @@ impl<'a, EK: KvEngine, ER: RaftEngine, T: Transport> StoreFsmDelegate<'a, EK, ER
         }
     }
 
+    fn report_min_resolved_ts(&self) {
+        let meta = self.ctx.store_meta.lock().unwrap();
+        let min_resolved_ts = meta.region_read_progress().get_min_resolved_ts();
```

Comment:
Here hold a global lock and iterator all regions. maybe have some problem, consider:

```
let meta = self.ctx.store_meta.lock().unwrap();
let read_progress = meta.region_read_progress().clone();
drop(meta)
let min_resolved_ts = read_progress.get_min_resolved_ts();
``` 
BTW, it's more better still running in pd-work if we can dispatch the config changes to it.

Reply:
Actually, I kept `task::ReportMinResolvedTs` which can be run in `pd-work`
I removed tick from `pd-work` because it's hard to support dynamic changes while there's a loop in a thread.